doc/compiled.json

```diff
@@ -21686,6 +21690,416 @@
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "title": "job_comment",
+                  "properties": {
+                    "id": {
+                      "type": "string"
+                    },
+                    "message": {
+                      "type": "string"
+                    },
+                    "job_id": {
+                      "type": "string"
+                    },
+                    "user": {
+                      "$ref": "#/components/schemas/user_preview"
+                    },
+                    "created_at": {
+                      "type": "string",
+                      "format": "date-time"
+                    },
+                    "updated_at": {
+                      "type": "string",
+                      "format": "date-time"
+                    },
+                    "mentioned_users": {
+                      "type": "array",
+                      "items": {
+                        "$ref": "#/components/schemas/user_preview"
+                      }
+                    }
+                  },
+                  "example": {
+                    "id": "abcd1234cdef1234abcd1234cdef1234",
+                    "message": "Some message...",
+                    "job_id": "abcd1234cdef1234abcd1234cdef1234",
+                    "user": {
+                      "id": "abcd1234cdef1234abcd1234cdef1234",
+                      "username": "joe.doe",
+                      "name": "Joe Doe"
+                    },
+                    "created_at": "2015-01-28T09:52:53Z",
+                    "updated_at": "2015-01-28T09:52:53Z"
+                  }
+                }
+              }
+            },
+            "headers": {
+              "X-Rate-Limit-Limit": {
+                "$ref": "#/components/headers/X-Rate-Limit-Limit"
+              },
+              "X-Rate-Limit-Remaining": {
+                "$ref": "#/components/headers/X-Rate-Limit-Remaining"
+              },
+              "X-Rate-Limit-Reset": {
+                "$ref": "#/components/headers/X-Rate-Limit-Reset"
+              }
+            }
+          },
+          "400": {
+            "$ref": "#/components/responses/400"
+          },
+          "404": {
+            "$ref": "#/components/responses/404"
+          },
+          "429": {
+            "$ref": "#/components/responses/429"
+          }
+        },
+        "x-code-samples": [
+          {
+            "lang": "Curl",
+            "source": "curl \"https://api.phrase.com/v2/projects/:project_id/jobs/:job_id/comments\" \\\n  -u USERNAME_OR_ACCESS_TOKEN \\\n  -X POST \\\n  -d '{\"message\":\"Some message...\"}' \\\n  -H 'Content-Type: application/json'"
+          },
+          {
+            "lang": "CLI v2",
+            "source": "phrase comments create \\\n--project_id <project_id> \\\n--job_id <job_id> \\\n--data '{\"message\": \"Some message...\"}' \\\n--access_token <token>"
```

Comment:
Same here with the message. I'd suggest to build the CLI and check how the parameters look like just to be sure